### DISCUSSION TITLE
为啥我发电了54.32元却显示6.66元

### DISCUSSION THREAD

**Discussion by Enigma-Soul** [Meloong-Git/PCL#1483] [提问] [answered]
![image](https://user-images.githubusercontent.com/119654171/227080374-bf36cf11-f168-4205-8958-9339412f4ec3.png)
![image](https://user-images.githubusercontent.com/119654171/227080405-538fff93-07ae-434c-8442-0a53ee41fa51.png)

**Comment by so1ve** [comment]
最后一次是6.66

**Reply by Enigma-Soul** [reply]
明明是5.68啊

**Comment by MoYuan-CN** [comment]
此处显示的是 **档位发电** 金额,自选发电不计入
~~爱发电的锅,没办法~~

**Comment by WForst-Breeze** [comment, marked as answer]
当前发电是指您**本月**的发电量，您只需要累计发电达到相应档位就可以获得该档位的所有权益。所以不用太过在意～
